### DISCUSSION TITLE
The reason why libraries suck

### DISCUSSION THREAD

**Discussion by cpgb85** [replit/kaboom#866] [Ideas]
You use a library, learn it in and out, then they say they are done. libraries suck

**Comment by niceEli** [comment]
kaboom is no longer maintained by replit but a community made version of kaboom (thats fully compatible with replits version of kaboom) has released. you can access the repo here [github.com/marklovers/kaplay](github.com/marklovers/kaplay) and see the documentation here [https://kaplayjs.com/](https://kaplayjs.com/). hopefully this solves your issue 😄.

**Reply by cpgb85** [reply]
no it does not. this is the issue with libraries.

**Reply by niceEli** [reply]
it is easy to migrate from kaboom to kaplay using this tutorial [https://kaplayjs.com/guides/migration-kaplay/](https://kaplayjs.com/guides/migration-kaplay/). the community of developers (including me and some of the original developers from replit) spent some time working on the fork of kaboom and making it as seamless as possible. depending on your circumstance it can take as little as changing 2 lines of code to migrate. the reason the library was discontinued wasnt a fault of the KAPLAY team or the original kaboom team, as the reason was because of replit downsizing to allow more money for AI to be spent. you can see info about why that happened here [https://github.com/marklovers/kaplay/wiki/The-relation-of-kaplay-with-Kaboom](https://github.com/marklovers/kaplay/wiki/The-relation-of-kaplay-with-Kaboom). hopefully this helps you continue your work with kaboom in its new home known as KAPLAY. 😄
